article-analysis-project/article-processing.js

```diff
@@ -0,0 +1,44 @@
+//Modules
+const unfluff = require('unfluff');
+const cheerio = require('cheerio');
+const request = require('request');
+//Variables and Consts
+const url = 'https://abcnews.go.com/Politics/wireStory/canceling-ivanka-trump-spurs-backlash-kansas-college-71174430?cid=clicksource_4380645_2_heads_hero_live_headlines_hed'
```

Comment:
Generally when I see a string like this in code it's an item that SHOULD be driven by configuration (and not checked into the repository).  I appreciate that this is R&D code so I would let it slide here, but better practice would be to use something like [dotenv](https://www.npmjs.com/package/dotenv) or [take in a parameter via command line](https://nodejs.org/en/knowledge/command-line/how-to-parse-command-line-arguments/) if it is a manually called script